Re-written Cumulocity Platform Introduction (#2466)
SoftwareAG/c8y-docs: content/concepts/concepts-introduction-bundle/analytics.md

---
weight: 50
title: Analytics
---

{{< product-c8y-iot >}} offers powerful analytics capabilities that enable businesses to extract valuable insights from their data as well as to operationalise derived insights using real-time rules. 

> For this, {{< product-c8y-iot >}} comes with [real-time data streaming and processing](/concepts/realtime/) capabilities, allowing businesses to process and take immediate action based on incoming data. Rules can be created within the [Streaming Analytics application](/streaming-analytics/introduction-analytics/#home) and the predefined [Smart Rules](/cockpit/smart-rules/) makes data analytics accessible also to non-technical users. Real-time rules do allow for both triggering external workflows (e.g. creation of a service ticket) as well as sending operations back to the equipment to e.g. optimize machine operations based on analytical insights.

Reviewer: ```suggestion
For this, {{< product-c8y-iot >}} comes with [real-time data streaming and processing](/concepts/realtime/) capabilities, allowing businesses to process and take immediate action based on incoming data. Rules can be created within the [Streaming Analytics application](/streaming-analytics/introduction-analytics/#home) and the predefined [Smart Rules](/cockpit/smart-rules/) makes data analytics accessible also to non-technical users. Real-time rules do allow for both triggering external workflows (for example, creation of a service ticket) as well as sending operations back to the equipment to, for example, optimize machine operations based on analytical insights.
```

> For this, {{< product-c8y-iot >}} comes with [real-time data streaming and processing](/concepts/realtime/) capabilities, allowing businesses to process and take immediate action based on incoming data. Rules can be created within the [Streaming Analytics application](/streaming-analytics/introduction-analytics/#home) and the predefined [Smart Rules](/cockpit/smart-rules/) makes data analytics accessible also to non-technical users. Real-time rules do allow for both triggering external workflows (for example, creation of a service ticket) as well as sending operations back to the equipment to, for example, optimize machine operations based on analytical insights.

Reviewer 2: ```suggestion
For this, {{< product-c8y-iot >}} comes with [real-time data streaming and processing](/concepts/realtime/) capabilities, allowing businesses to process and take immediate action based on incoming data. Rules can be created within the [Streaming Analytics application](/streaming-analytics/introduction-analytics/#home), and the predefined [smart rules](/cockpit/smart-rules/) make data analytics accessible also to non-technical users. Real-time rules do allow for both triggering external workflows (for example, creation of a service ticket) as well as sending operations back to the equipment to, for example, optimize machine operations based on analytical insights.
```

{{< product-c8y-iot >}} also offers the optional [DataHub](/datahub/datahub-overview/) application that is geared towards data analysts and data scientists and allows for offloading of IoT data to a data lake storage as well as to perform complex SQL queries on the data to derive deeper insights.

> 

Reviewer 3: DataHub mentioned twice. Here and under "IoT Data Management"

Author: Yes, this was on purpose as it is important for both data management and analytics. I will try to formulate it a bit differently so it doesn't look like a duplicate.

> The [{{< product-c8y-iot >}} DataHub](/datahub/datahub-overview/) makes it easy for data analysts and data scientists to derive insights from the equipment data by allowing for high-performant analytical queries. The support for SQL via standard ODBC/JDBC interfaces allow for a seamless integration with any data analytics tool and workflow.

Reviewer 2: ```suggestion
[{{< product-c8y-iot >}} DataHub](/datahub/datahub-overview/) makes it easy for data analysts and data scientists to derive insights from the equipment data by allowing for high-performant analytical queries. The support for SQL via standard ODBC/JDBC interfaces allow for a seamless integration with any data analytics tool and workflow.
```

> {{< product-c8y-iot >}} integrates seamlessly with popular analytics tools and AI frameworks, allowing businesses to apply sophisticated algorithms to their IoT data. For more information on machine learning capabilities, refer to the [Machine Learning Introduction](/machine-learning/ml-introduction/) in the documentation.

Reviewer: ```suggestion
{{< product-c8y-iot >}} integrates seamlessly with popular analytics tools and AI frameworks, allowing businesses to apply sophisticated algorithms to their IoT data. For more information on machine learning capabilities, refer to the [Machine Learning Introduction](/machine-learning/ml-introduction/) in the user documentation.
```
